test/app-tap/module_api.c

```diff
@@ -1075,7 +1075,7 @@ test_key_def_merge(struct lua_State *L)
 
 	/* Non-conventional prerequisite: list of known flags. */
 	uint32_t known_flags = key_part_def_known_flags();
-	assert(known_flags == BOX_KEY_PART_DEF_IS_NULLABLE);
+	assert(known_flags == (BOX_KEY_PART_DEF_IS_NULLABLE | BOX_KEY_PART_DEF_EXCLUDE_NULL));
```

Comment:
It is maybe a bit too formal, but we have test cases for merging (`box_key_def_merge()`) key parts, which differs only in the `is_nullable` flag (it is case 6 and 7). It looks logical to have the similar ones for the new `exclude_null` flag.

Reply:
Agree, more test cases is better.